src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+body,
+ul,
+ol,
+li {
+  font-size: 12px;
```

Comment:
Consider using relative units like `rem` or `em` for `font-size` instead of `px` to improve scalability and accessibility.